src/controlsListDomTokenList.js

```diff
@@ -0,0 +1,101 @@
+export default class DOMTokenList {
```

Comment:
If we think this class is general, we could just name the file DOMTokenList.js, since otherwise it associates to the controlsList property pretty strongly.

Reply:
Maybe the class isn't that general, please ignore

Reply:
> Maybe the class isn't that general, please ignore

Yes unfortunately not.